pkgutil completions are for Solaris CSW, let's avoid confusion with pkgutil on OS X
--- completions/Makefile.am.orig
+++ completions/Makefile.am
@@ -327,7 +327,6 @@
 		pkgadd \
 		pkgrm \
 		pkgtool \
-		pkgutil \
 		plague-client \
 		pm-hibernate \
 		pm-is-supported \
